src/adapters/controllers/patientLogin.controller.ts

```diff
@@ -0,0 +1,31 @@
+import PatientLoginUsecase from '@/domain/usecases/patientLogin/patientLogin.usecase';
+import { AuthPresenter } from '../presenters/auth.presenter';
+import { ErrorPresenter } from '../presenters/error.presenter';
+import { Controller, Request, Response } from './controller';
+
+export class PatientLoginController implements Controller {
+  constructor(private useCase: PatientLoginUsecase) { }
+
+  execute = async (input: Request): Promise<Response> => {
+    try {
+      if (!input.body) {
+        return {
+          body: undefined,
+          code: 400,
+        };
+      }
```

Comment:
man o que acha de colocarmos um validator error dentro do if ? ai o presenter iria capturar e formatar melhor o erro


```suggestion
      if (!input.body) {
        throw new ValidationError("body is empty")
      }
```